research-software-directory/RSD-as-a-service
# #214 Fix mentions components

# Fix mentions component to load colors from the theme

Closes #214 
Closes #218
Closes #62

Changes proposed in this pull request:
* The mentions section on software and project page uses secondary theme color. Previously the color was hard coded
* The change required refactoring on mui theme loading and updating of existing tests
* The Organization title in the header is changed to Organisation to be in line with other titles and labels currently used in RSD

How to test:
* `docker-compose down --volumes`: remove old instances
* `docker-compose build`: build solution (you could also build only frontend if you have most recent db structure)
* `docker-compose up`: start solution
* run data migration: cd data-migration && docker-compose up
* navigate to xenon software page with mentions, the mention section should look like this
![image](https://user-images.githubusercontent.com/9204081/163169728-5af0f518-85a5-468c-9332-851d89bc0fb5.png)
* navigate to 3D-e-Chem project and the impact and output section should look like this
![image](https://user-images.githubusercontent.com/9204081/163170111-167a3dad-ea64-46b8-a0da-fd11293484c0.png)

PR Checklist:
